### PR TITLE
ojs issue #2392 

### LINKED ISSUE #2392: OJS scraper does not work for https://tidsskrift.dk/sygdomogsamfund/issue/view/10555
**Describe the bug**
OJS scraper does not work for https://tidsskrift.dk/sygdomogsamfund/issue/view/10555

Nothing is scraped

**To Reproduce**
1. `python -m scholia.scrape.ojs issue-url-to-quickstatements https://tidsskrift.dk/sygdomogsamfund/issue/view/10555`
2. Nothing is printed

**Expected behavior**
Metadata should be printed to the terminal in the quickstatement format

### PR DESCRIPTION
**Pull request by faresh9**
## Title
Fixes #2392 

### Description
Updated the `issue_url_to_paper_urls` function to fix the issue where OJS scraper was not working for the provided URL (`https://tidsskrift.dk/sygdomogsamfund/issue/view/10555`). The function now uses a more robust XPath expression to extract paper URLs from the issue webpage.

### Caveats
No breaking changes introduced. No new dependencies added.

### Testing
Tested the updated function with the provided URL (`https://tidsskrift.dk/sygdomogsamfund/issue/view/10555`) and verified that paper URLs are successfully extracted.

### Checklist
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] My changes generate no new warnings
* [x] I have not used code from external sources without attribution
* [x] I have considered accessibility in my implementation
* [x] There are no remaining debug statements (print, console.log, ...)

### CODE REVIEW

File: scholia/scrape/ceurws.py [hunk 1]
```diff
@@ -405,4 +405,4 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    main()
```
Comment (fnielsen) [inline scholia/scrape/ceurws.py:408]: I do not understand this change. Are you removing the newline?

File: scholia/scrape/ojs.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def issue_url_to_paper_urls(url):
     URLs.
 
     """
-    response = requests.get(url, headers=HEADERS)
+    response = requests.get(url)
```
Comment (fnielsen) [inline scholia/scrape/ojs.py:85]: Why do you remove the headers argument with the HEADERS information?

File: scholia/scrape/ceurws.py (hunk 1 above)
Comment (fnielsen) [reply scholia/scrape/ceurws.py:408]: And why do you fix something that is not related to the issue?

File: scholia/scrape/ceurws.py [hunk 3]
```diff
@@ -357,21 +350,20 @@ def proceedings_url_to_proceedings(url, return_tree=False):
...
+    else:
+        acronym_elements = tree.xpath("//h1/a")
+        if len(acronym_elements) == 1:
+            proceedings['shortname'] = acronym_elements[0].text
 
-    # Add a check to ensure the list is not empty before accessing elements
-    urn_elements = tree.xpath("//span[@class='CEURURN']")
-    if urn_elements:
-        proceedings['urn'] = urn_elements[0].text
+    proceedings['urn'] = \
```
Comment (fnielsen) [inline scholia/scrape/ceurws.py:358]: Why do you remove the if line?

File: scholia/scrape/ceurws.py [hunk 4]
```diff
@@ -357,21 +350,20 @@ def proceedings_url_to_proceedings(url, return_tree=False):
...
+        tree.xpath("//span[@class='CEURURN']")[0].text
 
-    title_elements = tree.xpath("//span[@class='CEURFULLTITLE']")
-    if title_elements:
-        proceedings['title'] = re.sub(
-            r'\s+', ' ',
-            title_elements[0].text).strip()
+    proceedings['title'] = re.sub(
```
Comment (fnielsen) [inline scholia/scrape/ceurws.py:361]: Why do you remove the if line?

**Comment by fnielsen** [comment]
The PR has a too general title

File: scholia/scrape/ceurws.py (hunk 1 above)
Comment (faresh9) [reply scholia/scrape/ceurws.py:408]: my bad ceuws.py file should not be changed.

Comment (fnielsen) [reply scholia/scrape/ceurws.py:408]: Ok. Fine

**Pull request closed without merging**